src/logic/operate.js

```diff
@@ -0,0 +1,43 @@
+/*eslint-disable*/
+import Big from 'big.js';
+
+function operate(total, operation, next) {
+  let firstOperand;
+  let secondOperand;
+  try {
+    firstOperand = new Big(total);
+    secondOperand = new Big(next);
+  } catch (e) {
+    return 0;
+  }
+  let result;
+
+
+  switch (operation) {
+    case ('-'):
+      result = firstOperand.minus(secondOperand).toString();
+      break;
+
+    case ('+'):
+      result = firstOperand.add(secondOperand).toString();
+      break;
+
+    case ('%'):
+      result = firstOperand.times(secondOperand.div('100')).toString();
+      break;
```

Comment:
The percentage symbol should be a modifier like the `-/+` symbol, where `85% === 8.5`.

| Current behaviour | Recommended behaviour |
| :-----: | :-----: |
| ![percentage-bug](https://user-images.githubusercontent.com/36057474/77246109-4d510800-6c24-11ea-9297-44c5fb2b1bff.gif) | ![percentage-fix](https://user-images.githubusercontent.com/36057474/77246105-4c1fdb00-6c24-11ea-885c-cf5d3d76de95.gif) |



> `operate` switches `operation` and uses `plus`, `minus`, `times`, `div` and `modifier` (turning the number into a percentage based number, `e.g. 95 % == .95`) functions ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone4/performOperate.png))

Reply:
implemented it.